lib/browser.ts

```diff
@@ -188,7 +195,21 @@ export class ProtractorBrowser extends Webdriver {
    *
    * @type {boolean}
    */
-  ignoreSynchronization: boolean;
+  set ignoreSynchronization(value) {
```

Comment:
I'd also be OK with keeping these changes that keep `ignoreSynchronization` working but adding the new API as the 'recommended way' as well.